### PR TITLE
Tin can sealer.

### PR DESCRIPTION
**Pull request by Rivet-the-Zombie**
Adds a [tin can sealer](https://www.google.com/search?q=tin+can+sealer&ie=utf-8&oe=utf-8#q=tin+can+sealer+for+home+canning), allowing the player to produce sealed cans of meat, fish, human flesh,  vegetables, and fruit. One new item, five new recipes.

Changes the unused default container type for 'canned' meat, fish, human flesh, fruit, and vegetables (more info in [19c212c](https://github.com/Rivet-the-Zombie/Cataclysm-DDA/commit/19c212c91e7d1bb8728e3e9076bd7049636ab69e)) to accomplish this change.

### CODE REVIEW

**Comment by Darkyhard** [comment]
What about canned human flesh?

**Comment by Rivet-the-Zombie** [comment]
Good catch!

**Comment by Mecares** [comment]
Great now i only need a Rivtech atomic food irradiator to have access to all pre-cataclysm food preservation techniques.

File: data/json/items/comestibles.json [hunk 1]
```diff
@@ -13300,12 +13300,13 @@
...
         "id" : "apple_canned",
         "name" : "canned fruit",
+        "name_plural" : "canned fruit",
         "weight" : 450,
         "color" : "red",
         "addiction_type" : "none",
         "spoils_in" : 180,
         "stim" : 0,
-        "container" : "jar_glass",
+        "container" : "can_food",
```
Comment (Chezzo) [inline data/json/items/comestibles.json]: The description for just this item says, "Sealed glass jar containing preserved fruit.  Bland, mushy and losing color." 
 
All the rest are okay, though, I think.

**Comment by chaosvolt** [comment]
Ooh, nice. owo

**Comment by Rivet-the-Zombie** [comment]
>Great now i only need a Rivtech atomic food irradiator to have access to all pre-cataclysm food preservation techniques.

I may look into that when this PR is done.

**Comment by chaosvolt** [comment]
If it has good odds of leaking radiation maybe. X3

**Comment by Coolthulhu** [comment]
Wasn't food irradiation idea scrapped because reasons?
I think I recall Kevin commenting on it.

**Comment by Rivet-the-Zombie** [comment]
Yeah, now that I've thought on it for a while, I can't see it being conducted by anything small enough to carry on your person. Maybe some sort of fancy vehicle fixture that consumes plutonium, but not anything handheld.

**Comment by chaosvolt** [comment]
Hmm. This gives me one of those "sanity check" moments. How common is UV sterilization instead, for uses other than water purification?

**Comment by kevingranade** [comment]
Lots, but in industrial food processing, never at a small scale.

**Comment by chaosvolt** [comment]
As I guessed, Though recalling that cloudy water takes longer for a UV purifier to handle had me belatedly facepalming after typing that question. >.<

**Comment by Malkeus** [comment]
Walmart uses UV sterilization for the water dispenser.

**Comment by chaosvolt** [comment]
Though that's still water, and not exactly muddy creek water, despite how questionable some things from Walmart can be. XP

**Comment by Malkeus** [comment]
I think it's more to keep the nozzle sterile than to sterilize the water, tbh.

**Comment by kevingranade** [comment]
Wait, UV? That is not irridation. Irridation is heavy-duty radioactive
exposure. UV doesn't work on food because it doesn't penetrate.

**Comment by chaosvolt** [comment]
Again, did I mention the facepalming that ensued after I initial asked that question? :V

**Comment by Coolthulhu** [comment]
Minor problems:
* Requires a lot of tools to use - can sealer and hacksaw aren't easy to get early on
* Canning fruits halves the nutrition from them. Canning fish increases total nutrition by ~20%
* That can sealer is HUGE

**Comment by Rivet-the-Zombie** [comment]
>Requires a lot of tools to use - can sealer and hacksaw aren't easy to get early on

This was never intended to be an early game thing.

>Canning fruits halves the nutrition from them. Canning fish increases total nutrition by ~20%

Yeah, all that stuff really needs to be separately audited.

>That can sealer is HUGE

Did you check out the link I provided? They're pretty massive machines.

**Pull request merged**